src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -176,7 +178,7 @@ export default class FileField extends React.Component {
                   )}
                   {!hasErrors &&
                     _.get('properties.attachmentId', itemSchema) && (
-                      <div className="schemaform-file-attachment">
+                      <Tag className="schemaform-file-attachment review">
```

Comment:
If the `review` class is only being added to give the test a selector, can `schemaform-file-attachment` be used instead?

Reply:
The `review` class was added because on the review-and-submit page, a `review-row` without that wrapper will  not render the `dt` and `dd` properly.

Reply:
Gotcha.